### PR TITLE
Add context to _processIntent call from Launch of Skill

### PR DESCRIPTION
**Pull request by chadevanssf**

### CODE REVIEW

**Comment by chadevanssf** [comment]
Addresses https://github.com/salesforce/violet-conversations/issues/3

File: lib/conversationEngine.js [hunk 1]
```diff
@@ -585,7 +586,15 @@ class ConversationEngine {
...
   };
 
-
+  /**
+   * Override the default launch phrases. Parameter is used directly
+   * with the response.say() function when Alexa launches this app
+   * 
+   * @param {string[]} phrases - response or array of potential responses
+   */
+  setLuanchPhrases(phrases) {
```
Comment (muenzpraeger) [inline lib/conversationEngine.js]: Typo, should be setLaunchPhrases

Comment (vineet-sinha) [reply lib/conversationEngine.js]: @muenzpraeger - great catch. I am going to accept the PR and make the changes.

**Comment by vineet-sinha** [comment]
Thanks @chadevanssf!

**Comment by vineet-sinha** [comment]
Addresses #3 *and* parts of #5

**Pull request merged**